src/example/routes/index.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue'
+import Router from 'vue-router'
+
+// import Home from '../components/home'
+// const Home = () => import('../components/home')
```

Comment:
I just want to ask: why simple import is not enough [here](https://github.com/nuxt/nuxt.js/blob/dev/examples/dynamic-components/pages/index.vue#L16).
I do not understand the difference, i have read [this](https://vuejsdevelopers.com/2017/07/17/vue-js-2-4-important-features/) too.
Thank You in advance for your answer.

Reply:
`import Home from '../components/home'` 
Means that you import Home code in the bundle. 

`const Home = () => import('../components/home')`
Means you tell webpack to split it to another file (its own bundle) and lazy load it on demand.

Why ? because when someone load your first app view, you do not display all routes, only one, so why would you load other routes ? just load them on demand :)


Check this very cool [video ](https://www.youtube.com/watch?v=rn97hCNQsKI), and [here](https://router.vuejs.org/en/advanced/lazy-loading.html) for the docs.

Is it answering your question ?

Reply:
Yes, thank You. It is clearer.